charts/gha-runner-scale-set/values.yaml

```diff
@@ -47,6 +47,20 @@ githubConfigSecret:
 ## name of the runner scale set to create.  Defaults to the helm release name
 # runnerScaleSetName: ""
 
+## A self-signed CA certificate for communication with the GitHub server can be
+## provided using a config map key selector. If provided, the certificate will
+## also be mounted on runner pods under the path specified with
+## `runnerMountPath` (defaults to /usr/local/share/ca-certificates/ if left
+## blank) and NODE_EXTRA_CA_CERTS environment variable will be set to the path
```

Comment:
```suggestion
## `runnerMountPath` (defaults to /usr/local/share/ca-certificates/)
## and NODE_EXTRA_CA_CERTS environment variable will be set to the path
```

Reply:
I realise now this comment does not apply anymore, needs to be updated. If `runnerMountPath` is not provided we don't touch the runner pod spec at all.